.github/workflows/on-pr-default.yml

```diff
@@ -0,0 +1,21 @@
+# This file fires for the same events as `on-pr.yml`,
+# except where `on-pr.yml` skips over changes in paths-ignore,
+# this event fires on the inverse.
+# This is an officially blessed pattern for handling skipped but
+# required status checks. See 
+# https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/defining-the-mergeability-of-pull-requests/troubleshooting-required-status-checks#handling-skipped-but-required-checks
+# for more information.
+name: Pull Request
+
+on:
+  pull_request:
+    paths:
+      - '**' # Run if any file changes…
```

Comment:
Not sure if the logic or the comments are wrong but shouldn't this be "Run if only .version changes"?

Reply:
Did you get a peak at the following line? Sorry if that was unclear.

Reply:
LOL! I fixed this change in my test repo but didn't reflect it on this PR.